src/maintainer/adding_pkgs.rst

```diff
@@ -3,71 +3,94 @@
-   recipes produced by ``grayskull`` might need to be reviewed and edited. Read more about ``grayskull`` and how to use it `here <https://github.com/conda-incubator/grayskull#introduction>`__.
+#. If it's none of the above, generate a recipe with the help of a copy of `the example recipe <https://github.com/conda-forge/staged-recipes/tree/master/recipes/example>`_ in the `staged-recipes repository <https://github.com/conda-forge/staged-recipes>`_ and modify it as necessary.
 
 Your final recipe should have no comments (unless they're actually relevant to the recipe, and not generic instruction comments), and follow the order in the example.
 
 .. note::
 
   If there are any details you are not sure about please create a pull request anyway. The conda-forge team will review it and help you make changes to it.
 
-
-In case you are building your first recipe using conda-forge, a step-by-step instruction and checklist that will help you with a successful build is provided below.
-
 .. _staging_steps:
 
-Step-by-step Instructions
--------------------------
-
-#. Ensure your source code can be downloaded as a single file. Source code
-   should be downloadable as an archive (.tar.gz, .zip, .tar.bz2, .tar.xz)
-   or tagged on GitHub, to ensure that it can be verified. (For further
-   detail, see :ref:`tarballs_no_repos`).
-#. Fork the `example recipes
-   <https://github.com/conda-forge/staged-recipes/tree/master/recipes>`_
-   repository.
-#. Create a new branch from the staged-recipes ``master`` branch.
-#. Within your forked copy, generate a new folder in the recipes subdirectory
-   and copy the `meta.yml
-   <https://github.com/conda-forge/staged-recipes/blob/master/recipes/
-   example/meta.yaml>`_
-   file from the example directory. Please leave the example directory
-   unchanged!
-#. Edit the copied recipe (meta.yml) as needed. For details, see
-   :ref:`meta_yaml`.
-#. Generate the SHA256 key for your source code archive, as described in the
-   example recipe using the ``openssl`` tool. As an alternative, you can also
-   go to the package description on `PyPi <https://pypi.org>`_ from which you
-   can directly copy the SHA256.
-#. Be sure to fill in the ``test`` section. The simplest test will simply
-   test that the module can be imported, as described in the example.
-#. Remove all irrelevant comments in the ``meta.yaml``  file.
+.. tabs::
+   .. tab:: CRAN
+
+     #. If it is an R package from `CRAN <https://cran.r-project.org/>`_, kindly
+     start by using the `conda-forge helper script for R recipes <https://github.com/bgruening/conda_r_skeleton_helper>`_ instead.
+     Then if necessary, you can make manual edits to the recipe.
+
+   .. tab:: Grayskull
+
+
+      Generate a python package recipe with Grayskull:
+      --------------------------------------------------
```

Comment:
If you could make this header underline and the one in the other tabbed section the same length as the header, that would be great.